src/models/flight.js

```diff
@@ -78,7 +78,7 @@ let FlightModel = {
       return {
         id: flight.id,
         date: flight.date.substring(0, 10),
-        time: flight.time ? ' ' + flight.time.substring(0, 5) : '',
```

Comment:
Just passing time without extra space. Usually it's a view responsibly to decide how it needs to display time. I added `Util.formatDateAndTime` function that formats date and time and adds space in between